networking/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,82 @@ These release notes track the development of the Network Observability Operator
 
 For an overview of the Network Observability Operator, see xref:../../networking/network_observability/network-observability-overview.adoc#dependency-network-observability[About Network Observability Operator].
 
+[id="network-observability-operator-release-notes-1-4"]
+== Network Observability Operator 1.4.0
+The following advisory is available for the Network Observability Operator 1.4.0:
+
+* https://errata.devel.redhat.com/advisory/116729
+ https://access.redhat.com/errata/RHSA-2023:1817[RHSA-2023:1817 Network Observability Operator 1.4.0]
+
+[id="network-observability-channel-removal-1.4"]
+=== Channel removal
+You must switch your channel from `v1.0.x` to `stable` to receive the latest Operator updates. The `v1.0.x` channel is now removed.
+
+[id="network-observability-operator-1.4.0-features-enhancements"]
+=== New features and enhancements
+
+[id="network-observability-without-loki-1.4"]
+==== Network Observability without Loki
+You can now install the Network Observability Operator without Loki and only export flows to KAFKA or IPFIX. For more information, see xref:../../networking/network_observability/installing-operators.adoc#network-observability-without-loki_network_observability[Network Observability without Loki]. 
+
+[id="network-observability-enhanced-configuration-and-ui-1.4"]
+==== Notable enhancements
+The 1.4 release of the Network Observability Operator adds improvements and new capabilities to the {product-title} web console plugin and the Operator configuration. 
+
+This release introduces the following web console enhancements:
+
+* In the *Query Options*, the *Duplicate flows* checkbox is added to choose whether or not to show duplicated flows.
+* You can now filter source and destination traffic with image:arrow-up-long-solid.png[,10] *One-way*, image:arrow-up-long-solid.png[,10] image:arrow-down-long-solid.png[,10] *Back-and-forth*, and *Swap* filters. 
+* The Network Observability metrics dashboards in *Observe* -> *Dashboard* -> *NetObserv* and *NetObserv / Health* are modified as follows:
+** The *NetObserv* dashboard shows top bytes, packets sent, packets received per nodes, namespaces, workload, and pods. Flow graphs are removed from this dashboard.
+** The *NetObserv / Health* dashboard shows flows overhead as well as top flow rates per nodes, namespaces, workload, and pods.
+** Infrastructure and Application metrics are shown in a split-view for namespace, pod, and workloads. 
```

Comment:
```suggestion
** Infrastructure and Application metrics are shown in a split-view for namespace and workloads. 
```

Reply:
@jpinsonneau On all the namespace, workload, node, is it ok for all these to be plural, like namespaces, workloads, and nodes.

Reply:
yes sure :+1: